Makefile

```diff
@@ -15,10 +15,10 @@ venv:
 	[ -e ./venv/bin/pip-sync ] || ./venv/bin/pip install pip-tools
 	./venv/bin/pip-sync requirements.txt requirements-dev.txt
 
-.DEFAULT_GOAL := setup-dev
 .PHONY: setup-dev
 setup-dev: venv ${PRECOMMIT_DST}
 
+.DEFAULT_GOAL := lint
```

Comment:
I found myself wanting to run `make` to uhh... `make` sure that everything's ready to go. So let's make that the default.